src/assert.rs

```diff
@@ -397,8 +414,16 @@ impl Assert {
     /// ```
     pub fn unwrap(self) {
         if let Err(err) = self.execute() {
-            panic!("{}", err.display_chain());
+            panic!(Self::format_causes(err.causes()));
+        }
+    }
+
+    fn format_causes(mut causes: failure::Causes) -> String {
+        let mut result = causes.next().expect("an error should exist").to_string();
```

Comment:
Beside the name being off as-is, this is why I prefer `causes` not returning the current `Fail`. 
- Its always there
- Normally you print the first one differently.  The only clean way to do that with `Fail` is using `itertools::join` but I hesitated about adding it as a dependency (and it'd still look weird).